pkg/provider/infisical/provider.go

```diff
@@ -95,13 +95,18 @@ func (p *Provider) NewClient(ctx context.Context, store esv1beta1.GenericStore,
 			return nil, fmt.Errorf("failed to authenticate via universal auth %w", err)
 		}
 
+		secretPath := infisicalSpec.SecretsScope.SecretsPath
+		if secretPath == "" {
+			secretPath = "/"
```

Comment:
Won't this break existing things? Like, if you expected this path not to be set previously, it will be defaulted now to `/`.

Reply:
I need to recall why exactly I added this, but I think it was necessary?

I believe ExternalSecrets didn't work at all if you pass a secret name that includes a `/`, so you can always assume the only secrets that worked were from the root (`/`) and I believe this cannot be simply `""` when making an API request but I'm doing a quick pass to figure out why.

Reply:
Hmm I guess that's not necessarily true as `getSecretAddress` will return `("", "FOOBAR")` when the path is empty or has a `/`. I'll have to deploy this and test it.

Reply:
I verified the error. 👍